app/javascript/dashboard/routes/dashboard/contacts/components/AddReminder.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <div class="wrap">
+    <div
+      class="add-button"
+      :title="$t('REMINDER.ADD_BUTTON.TITLE')"
+      @click="onAdd"
+    >
+      <i class="ion-android-add-circle" />
+    </div>
+    <div class="input-select-wrap">
+      <div class="input">
+        <textarea
+          v-model="content"
+          class="input--reminder"
+          @keydown.enter.shift.exact="onAdd"
+        >
+        </textarea>
+      </div>
+      <div class="select">
+        <div class="date-wrap">
+          <i class="icon ion-android-calendar" />
+          <input
+            v-model="date"
+            :placeholder="$t('REMINDER.FOOTER.DUE_DATE')"
+            class="date-input"
+          />
+        </div>
+        <div class="label-wrap">
+          <select class="label" @change="optionSelected($event)">
```

Comment:
The naming `label` is misleading. Can we use task related name here `task__type` for this and below class names?